cmd/clusterctl/cmd/config_repositories_test.go

```diff
@@ -22,6 +22,7 @@ import (
 	"os"
 	"path/filepath"
 	"testing"
+        "fmt"
```

Comment:
I think the build is failing because this is now an unused import.

Reply:
:man_facepalming: